### PR TITLE
Backspace logic in v2 keypad: part 2

### PR DESCRIPTION
**Pull request by handeyeco**
## Summary:
I decided that since:
- The key translator can be customized by overwriting key->callback pairs, that it makes sense to just take the key handlers and have them be the defaults for the key translator. The consumer of the translator can customize as needed.
- The handlers are now mainly used by the translator, the logic should be together.

The core of this ticket though is backspace. Well I thought it would be fairly simple to take the backspace logic used by MathInput and move it to the translator since the backspace handler only takes a MathField object. However when testing I realized the logic couldn't be ported as-is. I tried investigating for a few seconds, but decided that it's not really a high priority as long as backspace works in desktop.

Filed a followup ticket: LC-906

Issue: LC-902

https://github.com/Khan/perseus/assets/16308368/3e723451-7595-4362-9e68-1d8019b5e89e

## Test plan:
- In the v2 keypad/MathQuill experience, write some stuff in the input (either typing or with the keypad)
- Use the backspace button to delete things
- Note the things are deleted

### CODE REVIEW

File: packages/math-input/src/components/input/math-wrapper.ts [hunk 1]
```diff
@@ -36,40 +33,15 @@ import {
...
 } from "./mathquill-types";
 
-function buildNormalFunctionCallback(command: string) {
-    return function (mathField: MathFieldInterface) {
-        mathField.write(`\\${command}\\left(\\right)`);
-        mathField.keystroke("Left");
-    };
-}
-
 const customKeyTranslator: Record<Key, MathQuillUpdaterCallback> = {
```
Comment (jeremywiebe) [inline packages/math-input/src/components/input/math-wrapper.ts:36]: So is `customKeyTranslator` the mobile instance of a key translator?  Could we name it as such (even in a future PR)?   Right now `custom` doesn't hit at why or what it's used for.

File: packages/math-input/src/components/input/math-wrapper.ts [hunk 2]
```diff
@@ -36,40 +33,15 @@ import {
...
-    JUMP_INTO_NUMERATOR: handleJumpOut,
-    JUMP_OUT_NUMERATOR: handleJumpOut,
-    JUMP_OUT_DENOMINATOR: handleJumpOut,
-    LEFT: handleArrow,
-    RIGHT: handleArrow,
-    LOG: buildNormalFunctionCallback("log"),
-    LN: buildNormalFunctionCallback("ln"),
-    SIN: buildNormalFunctionCallback("sin"),
-    COS: buildNormalFunctionCallback("cos"),
-    TAN: buildNormalFunctionCallback("tan"),
```
Comment (jeremywiebe) [inline packages/math-input/src/components/input/math-wrapper.ts:72]: Cool that we can pull this into the default translator!!

File: packages/math-input/src/components/key-handlers/key-translator.ts [hunk 3]
```diff
@@ -82,6 +106,10 @@ const keyToMathquillMap: Record<Key, MathQuillUpdaterCallback> = {
         mathQuill.keystroke("Left"); // under the root
     },
 
+    FRAC: (mathQuill) => {
+        mathQuill.cmd("\\frac");
+    },
```
Comment (jeremywiebe) [inline packages/math-input/src/components/key-handlers/key-translator.ts:111]: Can this just use `buildGenericCallback("\\frac", ActionType.CMD)`?

**Review by jeremywiebe — APPROVED** [review]
Cool!  Looks good.

File: packages/math-input/src/components/key-handlers/key-translator.ts (hunk 3 above)
Comment (handeyeco) [reply packages/math-input/src/components/key-handlers/key-translator.ts:111]: Good catch!

**Pull request merged**